static/ankersrv.css

```diff
@@ -117,3 +117,26 @@ code {
     border: 0 none;
     padding: 0;
 }
+
+.fullscreen-emulate {
+    position: fixed;
+    width: 90vw;
+    height: 99vh;
+    top: 1%;
+    left: 5%;
+    margin: 0;
+    padding: 0;
+    z-index: 9999;
+    overflow: hidden;
+    max-width: 90vw;
+    max-height: 99vh;
+}
+
+body.fullscreen-emulate {
+    background-color: rgba(0,0,0,0);
+}
+
+header.fullscreen-emulate,
+footer.fullscreen-emulate {
+    display: none;
```

Comment:
Maybe we should go the bootstrap way, and have a `.fullscreen-emulate-d-none` class, to mirror the `d-none` class from bootstrap?

Then we could have a generic css rule for `.fullscreen-emulate-d-none-active` here, and apply `.fullscreen-emulate-d-none-active` on `.fullscreen-emulate-d-none` from js? (see below)

Reply:
I'm not certain how that would improve functionality or the ability to follow what is happening with the change.

Reply:
Just an idea - we'll skip for now :+1: